web-browsers/browsers-and-dev-tools.md

```diff
@@ -0,0 +1,220 @@
+
+It also includes making API calls!
+
+Our fictional recipe blog might have the following behavior:
+
+- Listing ingredients that are needed based on the user inputting what's in their fridge
+- Scrolling to the top of the page slowly after the user clicks "Scroll to Top"
+- Rearranging recipes so that favorite recipes are listed first
+
+#### Possible Behaviors and the Web API
+
+A web browser keeps track of many things beyond rendering our websites. For example, browsers know where the mouse is positioned on the screen and they keep track of any [cookies](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies).
+
+## Introduction to Developer Tools
+
+Modern browsers provide a set of tools called Developer Tools, or Dev Tools.
+
+Dev tools are a set of web authoring and debugging tools. They can show us details as to how a specific website is _rendered_. With these tools, we'll be able to see the content and presentation of the site, and possibly the behavior.
+
+### !callout-info
+
+## Foreshadowing: Dev Tools Are for HTML and CSS development
+
+Dev Tools are useful when we dive into the details of HTML and CSS. But we haven't learned any concrete HTML or CSS problems! For now, the best thing we can do is get familiar with what the tools look like and anticipate what it can do for us in the future.
+
+### !end-callout
+
+### Accessing Dev Tools
+
+When we want to inspect the details of a website, we can open our Dev Tools using any of the following methods:
+
+- Right-click on a page and click on "Inspect" from the menu
+- Use the keyboard shortcut `cmd + opt + i`
+- Using the browser menu
+  - In Mozilla Firefox, we use Tools > Browser Tools > Web Developer Tools
+  - In Google Chrome, we use View > Developer > Developer Tools
+
+Once we do that, the Dev Tools show us details about the current page we're on.
+
+![Dev Tools opened on a web page. The web page is a lesson on the Learn platform.](../assets/web-browsers_browsers-and-dev-tools_dev-tools-intro.png)
+
+### Anatomy of the Dev Tools Window
+
+There are two main sections to navigating the Dev Tools window to focus on right now:
+
+1. The tabs, where each tab contains a different set of tools, all useful in different situations
+1. The content area, which shows the details of whatever tab is selected
+
+![Detailed, cropped view of the Dev Tools on the Inspector tab, showing the tabs and the content](../assets/web-browsers_browsers-and-dev-tools_dev-tools-window.png)
+
+| Tab Name               | Description                                                                         |
+| ---------------------- | ----------------------------------------------------------------------------------- |
+| Inspector or Elements  | Contains details on the rendered HTML elements                                      |
+| Console                | Has a JavaScript console that the current page has access to                        |
```

Comment:
turn this around, so that we have access to the page 
```suggestion
| Console                | Has a JavaScript console using which we can interact with the current page                                  |
```